aladdin.sh

```diff
@@ -113,7 +113,9 @@ function check_and_handle_init() {
             eval "$repo_login_command"
         fi
         local aladdin_image="$(jq -r '.aladdin.repo' "$ALADDIN_CONFIG_FILE"):$(jq -r '.aladdin.tag' "$ALADDIN_CONFIG_FILE")"
-        docker pull "$aladdin_image"
+        if [[ $aladdin_image == *"/"* ]]; then
+            docker pull "$aladdin_image"
```

Comment:
only do `docker pull` if the image is in the form: `repo/image:tag`, if there's no repo then don't do `docker pull`, maybe there's a better way to do this?